### PR TITLE
Create netlify.md

### PR DESCRIPTION
**Pull request by shashank1623**
added another cloud related application . that is free for use..

### CODE REVIEW

File: netlify.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Netlify
+description: Netlify is an all-in-one platform for automating modern web projects. Replace your hosting infrastructure, continuous integration, and deployment pipeline with a single workflow. Integrate dynamic functionality like serverless functions, user authentication, and form handling as your projects grow.
+created:  7 February 2014
```
Comment (onecompiler-ops) [inline netlify.md:4]: date format is wrong please use yyyy-mm-dd

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Comment by onecompiler-ops** [comment]
Could you please add cli commands too ?

**Comment by shashank1623** [comment]
> Could you please add cli commands too ?

for netlify supports drag and drop of your project dir and netlfiy algo will host your application. for the time being you merge my pr after that i will updated that if i i found cli commands

**Comment by shashank1623** [comment]
errors resolved now merge it!

**Comment by onecompiler-ops** [comment]
What is the value this cheat sheet adds? Why some one would refer to it?

**Comment by shashank1623** [comment]
in order if you want cli commands then i'm adding it. give a day

**Comment by shashank1623** [comment]
added the cli commands

**Comment by onecompiler-ops** [comment]
Screenshot 2022-10-04 at 11 06 55 AM
Markdown formatting broken, please fix it

**Comment by shashank1623** [comment]
issue solved!

**Review by onecompiler-ops — APPROVED** [review]

**Pull request merged**